src/sentry/incidents/models.py

```diff
@@ -469,8 +469,8 @@ class AlertRuleTrigger(Model):
     alert_rule = FlexibleForeignKey("sentry.AlertRule")
     label = models.TextField()
     threshold_type = models.SmallIntegerField()
-    alert_threshold = models.IntegerField()
-    resolve_threshold = models.IntegerField(null=True)
+    alert_threshold = models.FloatField()
+    resolve_threshold = models.FloatField(null=True)
```

Comment:
How precise would these values be? Will you run into float precision problems when you read these values back out or compare these values with other data? A DecimalField could help avoid those problems.

Reply:
I actually had these as `DecimalField` to being with but wasn't sure how useful it was considering we're passing floats from the frontend anyway. I suppose we could start passing them as strings.

I'm not sure we're going to have extremely precise values here - we won't be doing equality comparisons, and the values will likely be 2-4 decimal places at most. If you think it's worthwhile I'm good to change it though.

Reply:
Yeah we won't be doing math on these or anything like that, so I think we should be safe to use a float here.